### DISCUSSION TITLE
Alias is already in use

### DISCUSSION THREAD

**Discussion by mitko1725** [OrchardCMS/OrchardCore#9042] [Q&A] [answered]
Hello, I want to ask something regarding Alias.
Recently I was assigned a project which has
 Localization and menu with alias main-menu on both pages. (since I am using two Cultures.)
When I try to Publish or Draft something, I get the error "Your alias is already in use." 
![one](https://user-images.githubusercontent.com/58631213/113410985-5939e300-93bd-11eb-9a84-bc8f39dc84b6.PNG)

What should I do considering that on backend a menu is called using LINQ as you can see on provided photo.
![two](https://user-images.githubusercontent.com/58631213/113411156-bd5ca700-93bd-11eb-8a63-4acd7882e13f.PNG)

**Comment by Skrypt** [comment, marked as answer]
You can't use the same alias on 2 different content items. You will need to use a different alias simply.
